sormas-backend/src/main/java/de/symeda/sormas/backend/common/BaseAdoService.java

```diff
@@ -401,6 +403,13 @@ public boolean exists(ExistsPredicateBuilder<ADO> filterBuilder) {
 		}
 	}
 
+	public void incrementChangeDate(ADO ado) {
```

Comment:
Not sure whether "increment" is the perfect wording here, why not call it "updateChangeDate" or, even more explicit, "updateChangeDateToNow"?